.env.example

```
@@ -1,5 +1,9 @@
-GOOGLE_GENERATIVE_AI_API_KEY=
 API_URL=
 MONGO=
 
-NEXT_PUBLIC_ALLOW_ADS_MODE=true
\ No newline at end of file
+NEXT_PUBLIC_ALLOW_ADS_MODE=true
+NEXT_PUBLIC_PINO_LOG_LEVEL
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The NEXT_PUBLIC_PINO_LOG_LEVEL key should be with a value or have an equal sign

(KeyWithoutValue)